partcad/src/partcad/assembly.py

```
@@ -64,67 +58,63 @@ def add(
+            with pc_logging.Action("Assembly", self.project_name, self.name):
+                return await self._get_shape_real(ctx)
+        else:
+            return await self._get_shape_real(ctx)
+
+    async def _get_shape_real(self, ctx):
+        child_shapes = []
+
+        async def per_child(child):
+            # TODO(clairbee): use topods objects here
+            item = await child.item.get_build123d(ctx)
+            if not child.name is None:
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)